README.md

```diff
@@ -120,21 +185,35 @@ The following table gives the a description of the fields and corresponding valu
+The RNA-seq Alignment Workflow begins by trimming adapters, only if adapters are provided, using Cutadapt. 
+Sequencing quality is checked using FastQC and tumor/normal pairs are double-checked to confirm they are from the same individual using NGSCheckMate. 
+For more details on sample identity confirmation please see the [Kid's First NGS Checkmate Workflow](https://github.com/kids-first/ngs_checkmate_wf). 
+Reads were then aligned using STAR in two-pass mode to reference genome GRCh38. While all MTP data is paired-end, methods are provided for single-end alignment if you are interested in processing your data in the same manner. 
+Transcripts are quantified using RSEM at both the gene and isoform level  with the GENCODE v27 annotation, except for the GTEx samples which were not re-processed and are annotated using GENCODE v26. 
+Fusion calling is done using both Arriba and STAR-Fusion and then filtered for high confidence fusion calls using annoFuse. 
+QC metrics for the alignment are summarized using RNA-seQC. 
+If you would like to view the code in more detail, please see the GitHub release [Kids First RNA-seq Workflow](https://github.com/kids-first/kf-rnaseq-workflow) and if you would like to run the pipeline, please see the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-rnaseq-workflow/6). 
+Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation. 
 
 ### RNA Sequencing Data
 
 #### Fusions
 
-Gene fusions are called solely from RNA sequencing using the programs above. Fusions are filtered using custom R scripts. Fusion calls are retained if they are called by both STAR-Fusion and Arriba and if the fusion was specific and present in 3 or more samples in a single disease. Fusions were then annotated with gene and fusion specific information as well as whether they are known cancer genes from OncoKB, TCGA, and COSMIC. Summary frequencies are calculated using R. See [the fusion filtering documentation](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering) for specific code and further details. 
+Gene fusions are called solely from RNA sequencing using the programs above. 
+Fusions are filtered using custom R scripts. 
+Fusion calls are retained if they are called by both STAR-Fusion and Arriba and if the fusion was specific and present in 3 or more samples in a single disease. 
```

Comment:
This is also not true- 
```suggestion
We use annoFuse to filter out artifactual fusions, those with both gene partners having expression < 1 TPM, or fusions present in normal tissues. 
We annotate genes as transcription factor, tumor suppressor, oncogene, and/or kinase and for kinase genes, identify whether or not the kinase domain is retained in the fusion.
Fusions present in more than 4 broad histologies are also flagged and filtered out, as they may be non-specific.
This final file of putative oncogenic fusions is the data utilized for the fusion frequency tables.
```

See here: https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/dev/analyses/fusion_filtering/05-QC_putative_onco_fusion_distribution.Rmd